va/caa.go

```diff
@@ -39,24 +44,21 @@ func (va *ValidationAuthorityImpl) IsCAAValid(
 
 // checkCAA performs a CAA lookup & validation for the provided identifier. If
 // the CAA lookup & validation fail a problem is returned.
-func (va *ValidationAuthorityImpl) checkCAA(
-	ctx context.Context,
-	identifier core.AcmeIdentifier,
-	challengeType *string) *probs.ProblemDetails {
-	present, valid, records, err := va.checkCAARecords(ctx, identifier, challengeType)
+func (va *ValidationAuthorityImpl) checkCAA(ctx context.Context, ident core.AcmeIdentifier, params *caaParams) *probs.ProblemDetails {
```

Comment:
Can you keep this function signature spread across multiple lines? Its too long for one line. Similarly, I think leaving `ident` as `identifier` will remove some superfluous diffs.

Reply:
I've unwound this.

FWIW, in Go, wrapping of function definitions should generally be avoided and the need to do so suggests that there are potentially other issues with the coding style (see https://github.com/golang/go/wiki/CodeReviewComments#line-length).

In particular, wrapping in this style makes the function declaration bleed into the function code and I find that rather bad for readability (not to mention that wrapping and wrapping style is rather inconsistent in this code base :)